Automattic/pocket-casts-android
# Use shared debug key for signing debug app variants

## Description

For the top secret project (p1710978120584439-slack-C028JAG44VD) it will be easier to develop it if we all share the same debug signing key.

## Testing Instructions

> [!note]
> You need to have our app configured with release signing key from our secrets repository: PCYsg-iIC-p2

Assemble `debug`, `debugProd`, and `release` build type apps and verify signing certificate fingerprints. You can use [`apksigner`](https://developer.android.com/tools/apksigner) and `apksigner verify --print-certs <APK_PATH>` to print fingerprints.

### `debug` and `debugProd`

```
MD5:     7cb94785963b76f35ee4af0e0f2b638f
SHA-1:   fcd53d7c8a6befb8181f1d334afa0e245f1ac30c
SHA-256: a7a8cc75251243fcabc1c5ae6c2eaa3c74ad9a464ae4a5679e56820cbd9eadc8
```

### `release`

```
MD5:     c2fe0da38fcd86839b85da0837e63c08
SHA-1:   265bfda6992d75d455bc74dd43f9ace83efae7e5
SHA-256: ce0825f4ca785676d90d3af60f54fe43b8427afe9e1bb677add758dac1291820
```

## Checklist
 
#### I have tested any UI changes...
